monorepo-folders/package.json

```diff
@@ -17,6 +17,10 @@
     "concurrently": "^6.3.0"
   },
   "resolutions": {
+    "@temporalio/worker": "1.0.0",
+    "@temporalio/workflow": "1.0.0",
+    "@temporalio/client": "1.0.0",
+    "@temporalio/activity": "1.0.0",
```

Comment:
Is this fixed to `1.0.0`? Can we use the latest version?

Reply:
looks like it yes